frontend/src/component/integrations/IntegrationForm/IntegrationForm.tsx

```diff
@@ -142,7 +142,11 @@ export const IntegrationForm: VFC<IntegrationFormProps> = ({
                     : event.target.value;
             setFormValues(
                 produce((draft) => {
-                    draft.parameters[param] = value;
+                    if (value === undefined) {
```

Comment:
Solving:
```
src/component/integrations/IntegrationForm/IntegrationForm.tsx(145,21): error TS2322: Type 'string | undefined' is not assignable to type 'WritableDraft<{}>'.
  Type 'undefined' is not assignable to type 'WritableDraft<{}>'.
```

Reply:
@nunogois can you take a look if my change is safe? I tested it locally and it looks like parameter removal in addon still works fine and now TS is happy too

Reply:
Looks safe to me 👍